src/components/OpenSeadragonViewer.js

```diff
@@ -359,6 +367,7 @@ export class OpenSeadragonViewer extends Component {
     return (
       <section
         className={classNames(ns('osd-container'), classes.osdContainer)}
+        style={{ cursor: grabbing ? 'grabbing' : 'grab' }}
```

Comment:
Now that the `grab` style is upstream, we can just omit it here:

```suggestion
        style={{ cursor: grabbing ? 'grabbing' : undefined }}
```

Reply:
Done!
https://github.com/ProjectMirador/mirador/pull/3644/commits/891cc827d949ec7a3ea5d9a7f5559424726a281f